docs/ui/storage/menu.json

```diff
@@ -1,7 +1,6 @@
 {
     "title": "Storage",
     "items": [
-        "photo-picker",
```

Comment:
Oops, didnt see this, you actually dont need to remove this.

Reply:
Seeing an error when I add this back:
```
UnhandledPromiseRejectionWarning: Error: Ran into problems resolving "/docs/docs/ui/storage/photo-picker.md", referenced from "/docs/docs/ui/storage/menu.json"
```

There are no other framework fragments aside from React Native in `ui/storage`, so I think this should be safe to remove.

Reply:
Lets talk a look after stand up on this.

Reply:
as per our offline conversation, this is approved. Re-approving